ingests/faostat.py

```diff
@@ -163,8 +178,65 @@ def is_dataset_already_up_to_date(source_data_url, source_modification_date):
     return dataset_up_to_date
 
 
+class FAOAdditionalMetadata:
+    def __init__(self):
+        # Assign current date to additional metadata.
+        self.publication_date = str(dt.datetime.today().date())
+        self.publication_year = dt.datetime.today().date().year
+
+    @property
+    def metadata(self):
+        return {
```

Comment:
FYI you could also create `Dataset` object directly just like [here](https://github.com/owid/walden/blob/master/ingests/wb_income_groups.py#L28) to get type hints. It would warn you that `source_notes` field doesn't exist and doesn't get written into `faostat_metadata.json` (you could use a different field or we could create an `additional_info` field with `Any` type (dict, str, whatever).

Reply:
Thanks @Marigold , I'll do that. I have realised that, indeed, the `faostat_metadata.json` file added to the walden index does not have `source_notes`. But then I wonder, how is it possible that [the previous version of this file](https://github.com/owid/walden/blob/master/owid/walden/index/faostat/2022-02-10/faostat_metadata.json) does have `source_notes`?

Reply:
Oh, it doesn't matter, `Dataset` already has the field `access_notes`, which is exactly what I need.

Reply:
> But then I wonder, how is it possible that [the previous version of this file](https://github.com/owid/walden/blob/master/owid/walden/index/faostat/2022-02-10/faostat_metadata.json) does have source_notes?

No idea, maybe we've renamed the field to `access_notes`.